### PR TITLE
Update esm and electron mappings in preset-env

### PR DESCRIPTION
**Pull request by existentialism**
Guess I was too slow for @zloirock :)

Jokes aside, my original PR had a few other changes that are worth landing, so here they are post-rebase!

### CODE REVIEW

File: packages/babel-preset-env/data/built-ins.json [hunk 1]
```diff
@@ -215,10 +215,10 @@
     "edge": "12",
     "firefox": "5",
     "safari": "10.1",
-    "node": "10.0",
+    "node": "10",
```
Comment (nicolo-ribaudo) [inline packages/babel-preset-env/data/built-ins.json]: I think that `10.0` was better? Other features (e.g. `proposal-async-generator-functions`) have `node: 10.13`, so the minor version could be meaningful.

Comment (existentialism) [reply packages/babel-preset-env/data/built-ins.json]: For me, it’s about being consistent, so we don’t have multiple ways Of representing the same thing (`4.0` for one, and `4` for another).

I don’t find `10` vs `10.13` hard to decipher, in fact, I think that it makes minor versions _more_ noticeable.

But subjective stuff aside, thoughts on at least being consistent? Ultimately I’m fine with reverting this... not a huge deal :)

Comment (nicolo-ribaudo) [reply packages/babel-preset-env/data/built-ins.json]: If we handle `10` exactly as `10.0` it's ok like you prefer, I don't really care about it

File: packages/babel-preset-env/scripts/build-modules-support.js [hunk 2]
```diff
@@ -3,23 +3,22 @@ const fs = require("fs");
...
 const allowedBrowsers = {};
 
 Object.keys(stats).forEach(browser => {
-  if (browser !== "and_chr") {
+  if (!skipList.has(browser)) {
     const browserVersions = stats[browser];
     const allowedVersions = Object.keys(browserVersions)
       .filter(value => {
-        return acceptedWithCaveats[browser]
+        // Edge 16/17 are marked as "y #6"
```
Comment (danez) [inline packages/babel-preset-env/scripts/build-modules-support.js]: isn't that usually a sign it is not fully supported? or behind a flag?

Comment (existentialism) [reply packages/babel-preset-env/scripts/build-modules-support.js]: It's a bit more nuanced.

If it's not fully supported it's marked as `a #<footnote number`.

In this case, it is fully supported, except the caveat is that it loads the file, but doesn't execute it.

**Pull request merged**